templates/update.html

```diff
@@ -25,9 +25,12 @@ <h1>Update {{ mod.name }}</h1>
         <div class="row">
             <div class="col-md-4">
                 <h2 class="control-label">Mod version</h2>
+                <p>Previous: {{ mod.default_version.friendly_version }}</p>
             </div>
             <div class="col-md-8 form-group">
-                <input id="version" type="text" class="form-control input-block-level" placeholder="Version number" autofocus />
+                <input id="version" type="text" class="form-control input-block-level" placeholder="Version number" autofocus /><br>
+                <span id="version-warning" class="hidden text-warning"></span>
+                <button id="version-no" class="hidden btn-success">No!</button>
```

Comment:
I regret suggesting "No!" for this caption. :grimacing:
It should be something more positive that makes the user think they should click it.
How about...

```
Your last version had a \'v\' prefix, change this version to match? [ OK ]

Your last version had a \'V\' prefix, change this version to match? [ OK ]

Your last version didn\'t have a \'v\' prefix, change this version to match? [ OK ]

Your last version didn\'t have a \'V\' prefix, change this version to match? [ OK ]
```